apps/frontend/app/home/home.tsx

```diff
@@ -32,10 +150,56 @@ export function Home(): JSX.Element {
         </section>
 
         <section className={styles.shareableUrls}>
-          {/* TODO: Add shareableURls content here. The first bit of content has been set up for you. */}
-          <ShareIcon height={24} width={24} />
+          <ShareIcon height={90} width={90} className={styles.shareIcon} />
           <p>Shareable URLs</p>
-          <h2>Get instant feedback, the easy way</h2>
+          <h2>Get instant <br /> feedback, the <br /> easy way.</h2>
+          <div className={styles.slideContainer}>
+            <div className={styles[currentImage.className]}>
+              <img src={currentImage.src} alt="share a link" />
+            </div>
+            <div>
+              <div onMouseOver={handleProgressBar1} onFocus={handleProgressBar1} id={styles[progressBar1.active.toString()]}>
+                <div className={styles.progressBarContainer}>
+                  <div style={progressBar1Width}></div>
+                </div>
+                <LinkIcon height={27} width={27} />
+                <div className={styles.shareALinkContent}>
+                  <h2>Share a link</h2>
+                  <p>
+                    Forget sharing screenshots and copy-pasting code snippets.
+                    Get instant feedback on your code by sharing the URL of your sandbox with anyone.
+                  </p>
+                </div>
+              </div>
+              <div onMouseOver={handleProgressBar2} onFocus={handleProgressBar2} id={styles[progressBar2.active.toString()]}>
+                <div className={styles.progressBarContainer}>
+                  <div className={styles.pop} style={progressBar2Width}></div>
+                </div>
+                <ClickIcon height={27} width={27} />
+                <div className={styles.shareALinkContent}>
+                  <h2>Share the context</h2>
+                  <p>
+                    Those opening your sandbox will see your code running along with any tests,
+                    Storybook, or tasks you configured. No need to set up anything.
+                  </p>
+                </div>
+              </div>
+              <div onMouseOver={handleProgressBar3} onFocus={handleProgressBar3} id={styles[progressBar3.active.toString()]}>
+                <div className={styles.progressBarContainer}>
+                  <div style={progressBar3Width}></div>
+                </div>
+                <GroupIcon height={27} width={27} />
+                <div className={styles.shareALinkContent}>
+                  <h2>Powerful collaboration</h2>
+                  <p>
+                    Get async feedback through comments or host live coding
+                    sessions to learn together and guide others through the code.
+                  </p>
+                </div>
+              </div>
+              <Link href={'/s'} className={styles.getStarted}><h3>Get Started <ArrowIcon /></h3></Link>
```

Comment:
Use "#"  when the page that the link is supposed to point to doesn't yet exist 

```suggestion
              <Link href="#" className={styles.getStarted}><h3>Get Started <ArrowIcon /></h3></Link>
```